apps/marketplace/routes.js

```diff
@@ -73,6 +76,17 @@ export const Routes = () => (
       component={CloseOpportunitySuccessPage}
     />
     <PrivateRoute restrictedTo="buyer" path={`${rootPath}/brief/:briefId/overview`} component={BriefOverviewPage} />
+    <PrivateRoute
+      restrictedTo="buyer"
+      path={`${rootPath}/brief/:briefId/edit/:item?`}
+      component={EditOpportunityPage}
+    />
+    <PrivateRoute
+      restrictedTo="buyer"
+      path={`${rootPath}/brief/:briefId/edited`}
+      component={EditOpportunitySuccessPage}
+    />
+    <PrivateRoute restrictedTo="buyer" path={`${rootPath}/brief/:briefId/history`} component={OpportunityHistoryPage} />
```

Comment:
Do you think we should make this route similar to the public opportunity route? Eg. `digital-marketplace/opportunities/<id>/history`